packages/authorization-process/src/services/readModelService.ts

```diff
@@ -142,6 +142,41 @@ export function readModelServiceBuilder(
         ),
       };
     },
+    async getClientsRelatedToPurpose(
+      purposeId: PurposeId
+    ): Promise<Array<WithMetadata<Client>>> {
+      const data = await clients
+        .aggregate(
```

Comment:
we can use a simpler `find` in this case

Reply:
Updated

Reply:
Note: also `allowDiskUse` has been removed.